.eslint-plugin-local/index.ts

```diff
@@ -37,6 +37,9 @@ const schema = [
 	},
 ] as const;
 
+const REST_TYPE_NAME_REGEX =
+	/^REST(?:Get|Patch|Post|Put|Delete)\w+(?:JSONBody|FormDataBody|URLEncodedData|Result|Query)$/;
```

Comment:
`\w` includes `_`, which is not allowed in PascalCase
```suggestion
	/^REST(?:Get|Patch|Post|Put|Delete)[a-zA-Z0-9]+(?:JSONBody|FormDataBody|URLEncodedData|Result|Query)$/;
```